cli/build.sbt

```diff
@@ -2,5 +2,8 @@ libraryDependencies += "com.github.scopt" %% "scopt" % "3.7.0"
 
 libraryDependencies += "org.slf4j" % "slf4j-api" % "1.7.25"
 
-fork := true
+libraryDependencies += "io.circe" %% "circe-generic-extras" % "0.9.3"
```

Comment:
what is this for ?

Reply:
It is used in CliReads.scala to derive encoders/decoders with default values (for `RepoConfig`) and enumeration encoders/decoders for the sum type `TufServerType`.

Reply:
But isnt that already in `io.circe.generic` ? I mean that's what we use in all other projects ?

Reply:
No, `Configuration.default.withDefaults` and `deriveEnumerationEncoder`/`deriveEnumerationDecoder` are only in extras.

Reply:
There is `Decoder.enumDecoder` and the same for `Encoder` which we are using in other projects.